### DISCUSSION TITLE
docker运行失败

### DISCUSSION THREAD

**Discussion by long1318737396** [cubefs/cubefs#3099] [General]
git clone https://github.com/cubefs/cubefs.git
cd cubefs
make
成功之后
./docker/run_docker.sh -r -d /data/cubefs
发现就报错了
![image](https://github.com/cubefs/cubefs/assets/116941376/0ae4248a-a304-41eb-ad8c-c047bef98684)

**Comment by sejust** [comment]
Fix at da525628aba7a5590dc4f963175265574d8ecd97
